servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -250,8 +251,14 @@ public StreamingHttpClient buildStreaming() {
     }
 
     private static <U, R> StreamingHttpClient buildStreaming(final HttpClientBuildContext<U, R> ctx) {
-        final ReadOnlyHttpClientConfig roConfig = ctx.httpConfig().asReadOnly();
         final HttpExecutionContext executionContext = ctx.builder.executionContextBuilder.build();
+        // XXX This should probably be somewhere else
```

Comment:
Suggestions welcome as to how this should be handled. It seems that copying the tcp config, applying the configuration adjustment and then building the read-only config would be the most user-friendly approach.